test/conformance/kubetestgen/types.go

```diff
@@ -35,5 +35,7 @@ type Behavior struct {
 	APIObject   string `json:"apiObject,omitempty"`
 	APIField    string `json:"apiField,omitempty"`
 	APIType     string `json:"apiType,omitempty"`
+	Version     string `json:"version,omitempty"`
+	Status      string `json:"status,omitempty"`
```

Comment:
What is this field for? It doesn't seem to be used.

Reply:
The idea was to be able to have provisional behaviors that we know should be part of conformance but that we also know are not covered yet. In order to be conforming, a cluster needs to meet the required behaviors. That is, we want to be able to define the behaviors before the tests are ready.

I can add in a value for this for all the behaviors in this PR.

Reply:
ack. I'm worried this might introduce human error where we forget to remove this even after writing a test. Personally, I feel that the CI tooling should be able to determine this when generating the behavior <> test mapping. At the same time though, if it's defined here, coverage should be extremely easy to compute.